.gitlab-ci.yml

```diff
@@ -0,0 +1,18 @@
+stages:
+  - test
+
+noether-rocm:
+  stage: test
+  tags:
+    - rocm
+  image: jedbrown/rocm:latest
+  script:
+    - make -j$(nproc)
+    - echo '[{"subject":"/","metrics":[{"name":"Transfer Size (KB)","value":"19.5","desiredSize":"smaller"},{"name":"Speed Index","value":0,"desiredSize":"smaller"},{"name":"Total Score","value":92,"desiredSize":"larger"},{"name":"Requests","value":4,"desiredSize":"smaller"}]}]' > performance.json
```

Comment:
@jeremylt This is a placeholder that would show performance if we used GitLab Merge Requests. I'm hoping that like tests, it will eventually become available in the pipeline view.

Reply:
Would you mind if I throw a `make info` in here? It's in our other builds and I've found it useful for quick debugging in the past.

Reply:
Go for it.